scss/widgets/post.scss

```diff
@@ -120,6 +120,42 @@
     margin: 0 0 ($spacer * 0.5) 0;
     line-height: 1;
   }
+
+  .dc-topic-info {
+    margin-bottom: 0.5rem;
+
+    .dc-topic-category,
+    .dc-topic-tag {
+      color: $gray-800 !important;
+      margin-right: 0.5rem;
+      display: inline-block;
+    }
+
+    .dc-topic-category {
+      font-size: 0.9rem;
```

Comment:
Within `variables.scss` file we have:

```
$font-size-base
$font-size-lg
$font-size-sm
$font-size-xs
```

Keep using standard sizes whenever is possible. I see here `0.9rem` and then `0.8rem` maybe we can keep like: `$font-size-sm` and `$font-size-xs` respectively?

_Also the bottom line here is not about creating more variables for different sizes but to use a defined set of sizes to keep UI consistent_